### PR TITLE
Workaround due to SLEM rename

### PR DESCRIPTION
**Pull request by asmorodskyi**
VR: http://autobot.qe.nue2.suse.org/tests/4313

### CODE REVIEW

File: lib/publiccloud/azure.pm [hunk 1]
```diff
@@ -261,6 +262,7 @@ sub generate_azure_image_definition {
     return get_var('PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION') if (get_var('PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION'));
 
     my $distri = get_required_var('DISTRI');
+    $distri='sl-micro' if(is_sle_micro());
```
Comment (jlausuch) [inline lib/publiccloud/azure.pm:265]:
```suggestion
    $distri='sl-micro' if (is_sle_micro);
```

**Review by jlausuch — CHANGES_REQUESTED** [review]
Let's change it only when >=6.0

**Review by ricardobranco777 — APPROVED** [review]

File: lib/publiccloud/azure.pm [hunk 2]
```diff
@@ -261,6 +262,7 @@ sub generate_azure_image_definition {
     return get_var('PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION') if (get_var('PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION'));
 
     my $distri = get_required_var('DISTRI');
+    $distri = 'sl-micro' if (is_sle_micro('>=6.0'));
```
Comment (mimi1vx) [inline lib/publiccloud/azure.pm:265]: `sl-micro` ? isn't  correct value `sle-micro`

**Review by mimi1vx — CHANGES_REQUESTED** [review]

File: lib/publiccloud/azure.pm (hunk 2 above)
Comment (jlausuch) [reply lib/publiccloud/azure.pm:265]: The product has been renamed to `SUSE Linux Micro`, and this is what os-release's ID shows now.

Comment (mimi1vx) [reply lib/publiccloud/azure.pm:265]: district shouldn't be mixed with product so variable name is pretty misleading and here we should use `version_tools`

Comment (asmorodskyi) [reply lib/publiccloud/azure.pm:265]: the point is that distri variable used to generate expected image name and while for SLEM before version 6.0 image name contains `sle-micro` ( which matches with distri value ) , for 6.0 image name contains `sl-micro` .  And while I agree with your statements generally I can not follow what exactly you suggesting to address this ?

**Comment by asmorodskyi** [comment]
was already addressed by another PR

**Pull request closed without merging**